src/plugins/telemetry/public/services/telemetry_sender.test.ts

```diff
@@ -71,20 +71,20 @@ describe('TelemetrySender', () => {
       const telemetryService = mockTelemetryService();
       telemetryService.getIsOptedIn = jest.fn().mockReturnValue(false);
       const telemetrySender = new TelemetrySender(telemetryService);
-      const shouldSendRerpot = telemetrySender['shouldSendReport']();
+      const shouldSendReport = telemetrySender['shouldSendReport']();
 
       expect(telemetryService.getIsOptedIn).toBeCalledTimes(1);
-      expect(shouldSendRerpot).toBe(false);
+      expect(shouldSendReport).toBe(false);
     });
 
     it('returns true if lastReported is undefined', () => {
       const telemetryService = mockTelemetryService();
       telemetryService.getIsOptedIn = jest.fn().mockReturnValue(true);
       const telemetrySender = new TelemetrySender(telemetryService);
-      const shouldSendRerpot = telemetrySender['shouldSendReport']();
```

Comment:
I'm sure that was me with this faulty mac keyboard 😓

Reply:
Or me 😅 I have a spellchecker in my IDE because I'm always mixing up keystrokes... In mind, I know what I'm doing, but my fingers think otherwise... 🤦